app/Http/Controllers/User/ClientController.php

```diff
@@ -136,6 +124,33 @@ public function getClients(Request $request)
                         ->make(true);
     }
 
+    public function get2faLabel($active)
```

Comment:
only class name and title is different. Please create a common method and pass those as params

Reply:
Fixed